src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.cs

```diff
@@ -111,6 +112,11 @@ private PortableThreadPool()
             _legacy_minIOCompletionThreads = 1;
             _legacy_maxIOCompletionThreads = 1000;
 
+            if (NativeRuntimeEventSource.Log.IsEnabled())
+            {
+                NativeRuntimeEventSource.Log.ThreadPoolMinMaxThreads((ushort)_minThreads, (ushort)_maxThreads, (ushort)_legacy_minIOCompletionThreads, (ushort)_legacy_maxIOCompletionThreads);
```

Comment:
Nit: Would be nice to wrap long lines to make it easier to read, similarly elsewhere:
```suggestion
                NativeRuntimeEventSource.Log.ThreadPoolMinMaxThreads(
                    (ushort)_minThreads,
                    (ushort)_maxThreads,
                    (ushort)_legacy_minIOCompletionThreads,
                    (ushort)_legacy_maxIOCompletionThreads);
```